data/human/sales.txt

```diff
@@ -140,6 +140,48 @@ shipyard "Luxury Ships"
 	"Star Queen"
 	"Flivver"
 
+shipyard "Humanika Mimicry"
+	"Blackbird"
+	"Falcon"
+	"Bastion"
+	"Raven"
+	"Mule"
+	"Corvette"
+	"Leviathan"
+	"Behemoth"
+	"Bounder"
+	"Hauler"
+	"Falcon"
+	"Modified Argosy"
+	"Boxwing"
+	"Barb"
+	"Finch"
+	"Marauder Bounder"
+	"Nighthawk (New Tortuga)"
+	"Lance"
+	"Cutthroat"
+	"Mammoth"
+	"Bulk Freighter"
+
+
+outfitter "Humanika Mimicry"
+	"Capybara Reverse Thruster"
+	"Impala Plasma Thruster"
+	"Impala Plasma Steering"
+	"Liquid Nitrogen Cooler"
+	"S3 Thermionic"
+	"Tactical Scanner"
+	"Heavy Laser"
+	"Heavy Laser Turret"
+	"A120 Atomic Thruster"
+	"A125 Atomic Steering"
+	"A250 Atomic Thruster"
+	"A255 Atomic Steering"
+	"AR120 Reverse Thruster"
+	"Breeder Reactor"
+	"Heavy Anti-Missile Turret"
+	"Anti-Missile Turret"
+	"nGVF-DD Fuel Cell"
```

Comment:
An interesting line of future development, but I'd like to stick with local tech for now. Advanced New Tortuga engines seem like something the player would unlock after missions that gain their trust.

I think 200-series is reasonable, given that other places can make the 100-series. New Tortuga has advanced tech and poached employees, so a slight bump up in atomic engine tech is reasonable. A 300 series wouldn't be reasonable, which is why I stopped at 200. Likewise, with batteries and other categories, I didn't include the top tech, except for Syndicate. (They have mountains of Syndicate defectors.)

Reply:
I've added a comment explaining the lore. How does this look?

> `# Note on Humanika Mimicry.`
> `# For a few decades, New Tortuga has had advanced technology, Syndicate defectors, and a steady supply of poached employees. One of their top priorities has been technological advancement. They are only one planet though, so even with help from an advanced species, their success has been limited. Thus, this list is restricted low-to-mid-range tech from older companies. This is with the exception the the Syndicate, for which they already had access to top technology.`

Reply:
It is not a pirate world, and there are already job board jobs that explain the immediate attraction: the jobs are "lucrative." Most real-world employee poaching works by giving high pay and good job perks.

Next, the jobs give you access to alien technology to study. Some of these "poached employees" are double agents, feeding information back to the Syndicate or whoever else in return for trickles of information about human technology. Most are just fascinated by the opportunity to study something new.

If that were not enough, it is also a diverse religious planet, with a broad array of cultures in one place. (This is thanks to the Syndicate's oppression driving their people to New Tortuga.) Ethnic and religious diversity is enough to draw many people. I can justify that comment with personal experiences and stories of friends, but that's too long for github.

Lastly, there's the excitement of being on the frontier, in a planet that is nonetheless well protected. Local authorities have deals with pirates, to keep the New Tortuga cities safe. The presence of Korath keeps the skies free of unfriendly authorities.

Another attraction is the black markets. Anything illegal you can get in a pirate planet, you can get at New Tortuga, other than slaves.

Do you want to get high on deathsticks while studying Korath cooling systems?
How about attending a Hindu marriage ceremony for the first time, and seeing the groom ride in on an elephant?
Eat food from a different culture each day of the month?
Play 37 expensive holo-games that you didn't have to pay for?
Practice with grenades in a pirate-run shooting gallery?

You can do all of that on New Tortuga.

Reply:
I added this to the PR description.

Reply:
Indeed. I need to update the first paragraph of the description. Thank you for noticing that!

They do not broadcast the presence of alien technology to study, nor their abundance of employee poaching. That is something you have to look around for.

There are pirates on New Tortuga, so it does need to keep the `pirate` attribute.

Reply:
I have made revisions to address these issues. Do you like the results?

1. No 200-series atomic engines at New Tortuga.
2. Completely rewritten New Tortuga planet description.
3. Note in the txt that the `pirate` trait is because there are pirates present, and needs to be there for jobs.
4. News stories about New Tortuga residents enjoying the local culture and perks.

I'm keeping 100-series atomic engine level since other companies reproduced that level of tech.

Reply:
I have made more revisions to the "Humanika Mimicry" list and added comments explaining the rest. There were some ships in the list that had not been invented yet. Also, I added the Aerie and its pirate variant so there will be a second fighter carrier. The Aerie is described as being a thousand years old, so it is reasonable for New Tortuga to replicate it.

Reply:
Oh.. one more thing to note.

The Humanika governments can move freely about the Republic and trade with anyone, except the Syndicate. So, arranging an import agreement for low-end Deep technology is within their abilities. I think reverse engineering makes for a better story, but I could write a mutual agreement into it instead. Providing their thorough knowledge of Syndicate technology would be a good trade for a steady supply of lower-tech Deep Sky outfits and ships. That's boring, though.

Reply:
I think an angle of "receiving some but also, somewhat illicitly, building our own copies" could be an angle, as the campaign goes on, to explore both their Korath enhanced capabilities and their trade beyond the Syndicate, especially with how FW: Recon seems to hint at existing animosity between The Deep and Syndicate anyway.

Reply:
I disagree with adding the Aerie to their shipyard. While it could make sense as you explained, the Aerie is a Deep-specific vessel and important for Northern Pirate activity, and the Pirate Variant was made explicitly for the Northern Pirates - each Pirate Variant was made for their respective areas. There is a planned Pirate Variant ship for the Core Pirates, which is what would be used instead rather than importing something all the way from the other side of the map. I could *maybe* see them obtaining Aeries etc. *later* into the story, but as it stands that seems like too much on top of their already wide selection of technology (such as Atomics). There are future plans to expand Deep story too, so I would prefer we keep Deep-related content like that for them, and expand upon it later rather than placing it as a default in another area.

Reply:
Unless Pirate Variants are strictly necessary to this PR, I would also recommend not including them at this time. They may be in the game files, but they are not implemented the way we want them to be and will need another PR or so going over them and their stats. If they are essential to this PR, which it doesn't seem like that to me, I still think you should wait for them to be properly implemented before including them.

Reply:
Can you suggest another fighter carrier to add in its place?

Reply:
> Additionally, unless Pirate Variants are strictly necessary to this PR

The Nighthawks are critical to the story. Others are just for variety and environment.

Reply:
Is a fighter carrier that necessary, and if so, why?

Reply:
The ship chosen for the Core Pirate Pirate Variant was the Protector. I am of mind to make some sort of somewhat stripped yet edited Protector that instead of all the turret mounts has 2 or 4 bays in place of some of them. That could be used instead of an Aerie, and it would be more unique than just importing one, being it would be something only the Core Pirates have. I can't guarantee that's the build we choose, though, but it's the idea that's been thrown around.

I still recommend not basing things on Pirate Variants before we properly implement them, though. The lore should be flexible enough to not necessitate things like that, or ships such as fighter carriers.

Reply:
The more I think about it though, this PR might have to wait, and it would probably be beneficial to both parties, for the Pirate Campaign and expanded content related to it (more Pirate systems, Variants, etc.) to be reviewed/merged, because that may change the demographic of Pirates enough for this PR to need to adjust, and preferably this PR adjusts to Pirate-focused content and not vice versa.

Reply:
A stripped-yet-edited Protector is a *far* more interesting option than an Aerie. For now, the Aerie is my only justifiable option for a fighter carrier, so I'll have to keep it. Consider it a placeholder, if you will.

Fighters are a legitimate strategy, often employed by Endless Sky players. A basic fighter carrier should be available at a "smuggler's haven." They don't need top-of-the-line fighter carriers like the Bactrian, but they should have something simple.

Actually, a Protector may be too expensive for an early-game smuggler, at 1.5-2x the price of an Aerie. Not sure.

Reply:
> The more I think about it though, this PR might have to wait, and it would probably be beneficial to both parties, for the Pirate Campaign and expanded content related to it (more Pirate systems, Variants, etc.) to be reviewed/merged, because that may change the demographic of Pirates enough for this PR to need to adjust to it.

Development should continue in parallel, as a collaboration. However, if the Pirate Campaign turns out to be a year or two from having a viable PR, I'd like this one to go in first. (I don't know the status or progress, so please don't be offended if that is totally off-base.) Certainly, I'll make sure to deal with any known problems first.

Recall this PR is mainly a side-story, providing an optional intro to the Korath campaign. The critical parts (first contact and government changes) can be split into a separate PR. The Korath Translator story is to come next, and that is where the campaign actually begins. It'll be mostly self-contained, to avoid collisions with other campaigns.

To put it another way, I anticipated possible problems, and that is why the fish & feather is just a side-story, and not something critical to the campaign.

Reply:
One other thing though... The inability of Korath interact directly with ordinary Pirates or the Syndicate is *non-negotiable*. Those groups' actions and morals are fundamentally incompatible with the Korath Exiles of ravenshining's lore document. They need an intermediary, or it'll result in shooting.

Reply:
I see no problem replacing the Aerie with this new specific Protector variant when it's finalized as part of actually putting the Protector into the game, whether that be through Pirate campaign related PRs or an individual one.

Until then, a sensible placeholder meant for later replacement, like the Aerie, is entirely reasonable.